generate-samples.py

```diff
@@ -0,0 +1,95 @@
+from human_eval.data import write_jsonl, read_problems
+from openai import OpenAI
+
+import argparse
+import sys
+
+# This script consumes a .jsonl file that contains a JSON object, which has a
```

Comment:
A `.jsonl` file does not contain a JSON object.  It contains a list of JSON objects -- not a JSON list, but one after the other in the file.